crates/settings_ui/src/keybindings.rs

```diff
@@ -1239,7 +1246,19 @@ impl Focusable for KeystrokeInput {
 }
 
 impl Render for KeystrokeInput {
-    fn render(&mut self, _window: &mut Window, cx: &mut Context<Self>) -> impl IntoElement {
+    fn render(&mut self, window: &mut Window, cx: &mut Context<Self>) -> impl IntoElement {
+        if self.focus_handle.is_focused(window) {
```

Comment:
We probably want to do this in a dedicated focus listener, rather than waiting for the next frame to fix the state :)